src/SFA.DAS.Forecasting.Commitments.Functions/Startup.cs

```diff
@@ -133,7 +141,26 @@ public void ConfigureLogFactoy()
         {            
             _loggerFactory = new LoggerFactory();
             var logger = _loggerFactory.CreateLogger("Startup");            
-        }       
+        }
+
+        protected IDocumentSession CreateDocumentSession(IConfigurationRoot config)
+        {
+            var connectionString = config["CosmosDbConnectionString"];
+            if (string.IsNullOrEmpty(connectionString))
+                throw new InvalidOperationException("No 'DocumentConnectionString' connection string found.");
+            var documentConnectionString = new DocumentSessionConnectionString { ConnectionString = connectionString };
```

Comment:
should the connection string be passed through as a constructor param? The class is pretty useless without it.